.github/workflows/cs.yml

```diff
@@ -0,0 +1,41 @@
+name: Coding Standards
+
+on:
+  pull_request:
+
+jobs:
+  cs:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        php: [7.4]
+    steps:
+      - name: Cancel previous runs of this workflow (pull requests only)
+        if: ${{ github.event_name == 'pull_request' }}
+        uses: styfle/cancel-workflow-action@0.5.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          tools: composer:v2
+          coverage: none
+
+      - name: Install composer dependencies
+        uses: nick-invision/retry@v1
+        with:
+          timeout_minutes: 5
+          max_attempts: 5
+          command: composer install
```

Comment:
I'd recommend using https://github.com/ramsey/composer-install or https://github.com/php-actions/composer here.

I'm also not sure how likely it is that an action can timeout that would justify this action. Mainly because I have never seen this happen with Github Actions before. ¯\_(ツ)_/¯

Reply:
Done! Very helpful suggestion.